Microsoft.Toolkit.Parsers/Markdown/Inlines/ImageInline.cs

```diff
@@ -120,7 +118,7 @@ internal static InlineParseResult Parse(string markdown, int start, int end)
 
             if (pos < end && markdown[pos] == '[')
             {
-                refstart = pos;
+                var refstart = pos;
```

Comment:
I think this makes readability harder than `int refstart = pos;`

Reply:
I prefer using `var` instead of the "real" type. I've still reverted this change since it is not really related to this PR.